### ISSUE TITLE
default readme in BIDS dataset after conversion

### ISSUE THREAD

**Issue by Remi-Gau** [nipy/heudiconv#583]
hey

We (BIDS maintainers) are trying to see if we can improve the quality of the average README in BIDS dataset. Those tend to be pretty thin and are missing some information we would like to see there even if it can be formalized "yet" into the BIDS specification.

Hoping to leverage the bids converters to help to do.

So I am going around the different BIDS converters repo to see if we can start improving the  default README they put in a the BIDS dataset.

There is template that is provided in the BIDS starter kit: https://github.com/bids-standard/bids-starter-kit/blob/main/templates/README.MD

Some possible options:
1. use this template is
2. use a subset of it
3. add a link to the template README
4. something else (a mix of 2 and 3?)

What do you think? I can try to open a PR if you want.

Also let us know if you think that the template README in the starter kit could / should be improved .

Feel free to disregard if you are already using some version of the template README.

**Comment by Remi-Gau** [comment]
Given that heudiconv can also output datalad friendly datasets, maybe the readme could also include 

http://handbook.datalad.org/en/latest/basics/101-180-FAQ.html#how-can-i-help-others-get-started-with-a-shared-dataset

disregard if this is already the case

**Comment by yarikoptic** [comment]
Dear @Remi-Gau , here is where we are creating README.txt if it doesn't yet exist.  Probably we should switch to README.md (but not generate it if any form of README.* exists): https://github.com/nipy/heudiconv/blob/e747f0595dd393384d65219f5137d42b2a5c8e73/heudiconv/bids.py#L149 .  Interested to propose a PR with changes you think it should have ?

**Comment by Remi-Gau** [comment]
Yup I can try to PR this. 
Would it make sense to ship a template file with the package and copy or read from it rather than add a VERY long string in that bids.py file?

**Comment by yarikoptic** [comment]
yeah, if sizeable , lets add `heudiconv/bids/templates` - and dont' forget to add that to `package_data=` in setup.py. THANKS!
